pkgs/applications/editors/standardnotes/standardNotes.nix

```
@@ -0,0 +1,30 @@
+with import <nixpkgs> {};
+
+stdenv.mkDerivation {
+  name = "standardNotes-2.3.11";
+
+  src = fetchurl {
+    url = https://github.com/standardnotes/desktop/releases/download/v2.3.11/standard-notes-2.3.11-x86_64.AppImage;
+    sha256 = "1zbkmxsipmprrbgjmszg33w7cqs5v5zrcwknkqwybj9q41addsql";
+  };
+
+  buildInputs = [ appimage-run  ];
+
+  phases = " installPhase";
+
+   installPhase = ''
+    mkdir -p $out/bin
+    cp $src $out/standardNotes.AppImage
+    echo "#!usr/bin/env bash" > $out/bin/standardnotes
+    echo "${appimage-run}/bin/appimage-run $out/standardNotes.AppImage" >> $out/bin/standardnotes
+    chmod +x $out/bin/standardnotes $out/standardNotes.AppImage
+   '';
+
+
+  meta = {
+    description = "File Manager and User Shell for the GNU Project";
+    homepage = https://standardnotes.org;
+    license = "AGPL3.0";
+    maintainers = [ stdenv.lib.maintainers.mgregoire ];
```

```suggestion
    maintainers = with maintainers; [ mgregoire ];
```

Seems to be the convention

---

Indeed, thank you!